addons/escoria-core/game/core-scripts/esc/esc_globals_manager.gd

```diff
@@ -92,6 +92,15 @@ func set_global(key: String, value, ignore_reserved: bool = false) -> void:
 				"Global key %s is reserved and can not be overridden" % key
 			]
 		)
+	
+	if key.count("/") >= 2 and key.substr(1, 1) == "/":
```

Comment:
```suggestion
	if key.count("/") > 1 and key.begins_with("/"):
```

Reply:
You suggestion is incorrect: the "/" character can only be located at second position in the string, not as the first character.